### PR TITLE
Max UI branch Close #3

### LINKED ISSUE #3: PhotoFeedView-Max
UI for the photo feed.
DOD: A page that will have allocated space for displaying data cards.

### PR DESCRIPTION
**Pull request by MaxIgnatyev**
Not ready to merge, but available to review code so far.

### CODE REVIEW

File: photo-points/Views/Home/PhotoStream.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
+﻿<!DOCTYPE html>
```
Comment (weberwilliamb) [inline photo-points/Views/Home/PhotoStream.html]: Aren't these supposed to be cshtml?

**Review by weberwilliamb — APPROVED** [review]
Looks good to me.

**Review by RoEric — APPROVED** [review]
I think it looks good.

**Review by sruble — APPROVED** [review]
Looks good Max, I see that you added a bootstrap library.

**Review by Sara-webaplication — APPROVED** [review]

**Review by weberwilliamb — APPROVED** [review]
Looks good!

**Pull request merged**